### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by ntgussoni** [markdoc/markdoc#153] [General]
Hi guys, first of all, thank you for the work into this. I think it's very promising.

I'm currently making some work on top of markdoc, although it's good for most use cases, I find it hard to extend the most core features. 
Perhaps this is intended.

Is there any roadmap that you guys can share? I'd be glad to contribute to it

**Comment by mfix-stripe** [comment]
@ntgussoni we do have a public roadmap at the moment, but this is absolutely something we want to create and share with you folks 🙂 Once we do, we will share it as a GitHub Project.

**Comment by mfix-stripe** [comment]
Added our public roadmap here: https://github.com/orgs/markdoc/projects/1/views/1

Cc @markdoc/core
